x-pack/plugins/snapshot_restore/public/app/sections/home/home.tsx

```diff
@@ -121,7 +120,7 @@ export const SnapshotRestoreHome: React.FunctionComponent<RouteComponentProps<Ma
               onClick={() => onSectionChange(tab.id)}
               isSelected={tab.id === section}
               key={tab.id}
-              data-test-subject={tab.testSubj}
+              data-test-subj="tab"
```

Comment:
I prefer to be declarative on what an element _is_ instead of giving it a rigid ID. I find it more flexible and the tests cleaner. We have all the necessary helpers to then target the node we want.  

The same would apply for a menu for example. Keep it generic and then have all the flexibility in the tests to read the nodes and target one.

```js
const menuItems = find('myMenu.item'); // only 2 data-test-subj "myMenu" and "item"
expect(menuItems.map(item => item.text())).toEqual(['item1, 'item2', 'item3']);

menuItems.at(1).simulate('click');
```

We would have to update the test if an item is added to the menu, but that's also the point of the test, isn't it? 😊